components/AnalysisSummary.tsx

```diff
@@ -1,6 +1,172 @@
-import React from 'react'
+import { Paper, Table, TableBody, TableCell, TableContainer, TableHead, TableRow } from '@material-ui/core'
+import { format } from 'date-fns'
+import _ from 'lodash'
+import React, { useMemo } from 'react'
 
-import { Analysis, ExperimentFull, MetricBare } from '@/models'
+import {
+  Analysis,
+  AnalysisStrategyToHuman,
+  ExperimentFull,
+  MetricBare,
+  Recommendation,
+  RecommendationWarningToHuman,
+  Variation,
+} from '@/models'
+
+/**
+ * Convert a recommendation's endExperiment and chosenVariationId fields to a human-friendly description.
+ */
+function RecommendationString({
+  recommendation,
+  experiment,
+}: {
+  recommendation: Recommendation
+  experiment: ExperimentFull
+}) {
+  if (recommendation.endExperiment) {
+    if (recommendation.chosenVariationId) {
+      const chosenVariation = experiment.variations.find(
+        (variation) => variation.variationId === recommendation.chosenVariationId,
+      ) as Variation
+      return (
+        <>
+          End experiment; deploy <code>{chosenVariation.name}</code>
+        </>
+      )
+    }
+    return <>End experiment; deploy either variation</>
+  }
+  return <>Keep running</>
+}
+
+/**
+ * Render a table of participant counts based on the latest metric analyses for the given experiment.
+ */
+function ParticipantCounts({
+  experiment,
+  latestPrimaryMetricAnalyses,
+}: {
+  experiment: ExperimentFull
+  latestPrimaryMetricAnalyses: Analysis[]
+}) {
+  const sortedVariations = _.orderBy(experiment.variations, ['isPrimary', 'name'], ['desc', 'asc'])
+  return (
+    <TableContainer component={Paper}>
+      <Table>
+        <TableHead>
+          <TableRow>
+            <TableCell>Strategy</TableCell>
+            <TableCell>Total</TableCell>
+            {sortedVariations.map((variation) => (
+              <TableCell key={variation.variationId}>
+                <code>{variation.name}</code>
+              </TableCell>
+            ))}
+          </TableRow>
+        </TableHead>
+        <TableBody>
+          {latestPrimaryMetricAnalyses.map((analysis) => (
+            <TableRow key={analysis.analysisStrategy}>
+              <TableCell>{AnalysisStrategyToHuman[analysis.analysisStrategy]}</TableCell>
+              <TableCell>{analysis.participantStats.total}</TableCell>
+              {sortedVariations.map((variation) => (
```

Comment:
It would probably make it a bit more readable to deconstruct the variation id here and elsewhere:

```suggestion
              {sortedVariations.map(({variationId}) => (
```